docs/source/faq.rst

```diff
@@ -121,9 +121,7 @@ It is possible to customize the value that will be shown for {name} by using the
         }
         return "";
     }
-    doc.setOptions({
-        nullGetter: nullGetter,
-    });
+    const doc = new Docxtemplater(zip, {nullGetter: nullGetter,});
```

Comment:
Can you remove the trailing comma ?

    const doc = new Docxtemplater(zip, {nullGetter: nullGetter});

Reply:
(And the other trailing commas).

Reply:
sure.
Done in [this](https://github.com/shadab14meb346/docxtemplater/commit/a69056007c3295453aa441807d6e507f96230d7c) commit.